sql-grammar.md

```diff
@@ -17,4 +17,6 @@ a[name]:focus {
 }
 </style>
 
+{{site.data.alerts.callout_success}}This page describes the full CockroachDB SQL grammar. However, as a starting point, it's best to reference our <a href="sql-statements.html">statement-specific pages</a> first, which provide detailed explanations and examples.{{site.data.alerts.end}}
```

Comment:
Turns out that adding `markdown="1"` to the the relevant alert divs in `_data\alerts.yml` does let us use markdown formatting within alerts. One gotcha though is that the liquid before and after an alert needs to be on its own line. I'll update our internal docs on this.